update privacy readme for opSupportLimits (#808)
webmachinelearning/webnn: security-privacy.md

> - 2.6 What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?

> The [opSupportLimits()](https://www.w3.org/TR/webnn/#api-mlcontext-opsupportlimits) exposes op level feature support limits that are different across backends. The information exposed roughly correspond to the underlying operating system version so it shouldn't introduce more entropy. 

Reviewer: ```suggestion
The [opSupportLimits()](https://www.w3.org/TR/webnn/#api-mlcontext-opsupportlimits) exposes per-operator feature support limits that are different across backends. The information exposed roughly corresponds to the underlying operating system version, and so it shouldn't introduce more entropy.
```
Prefer to spell out full words in prose.

> Additionally an execution time analysis may reveal indirectly the performance of the underlying platform's neural network hardware acceleration capabilities relative to another underlying platform.

Author: thanks for the suggestion! sounds a lot better.
